var/spack/repos/builtin/packages/py-celery/package.py

```diff
@@ -46,19 +47,27 @@ class PyCelery(PythonPackage):
     # 'cosmosdbsql',
     # 's3',
 
+    depends_on('python@3.7:', type=('build', 'run'), when="@5.2.3:")
     depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
 
     depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-setuptools@59.1.1:59.6', type=('build', 'run'), when="@5.2.3:")
 
     depends_on('py-redis@3.2.0:', when='+redis', type=('build', 'run'))
+    depends_on('py-redis@3.4.1:3,4.0.2:', when=('+redis', '@5.2.3:'), type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('py-redis@3.4.1:3,4.0.2:', when='@5.2.3:+redis', type=('build', 'run'))
```
This is what's causing the tests to fail

Reply:
Updated